hudi-client/hudi-client-common/pom.xml

```diff
@@ -41,6 +41,11 @@
       <artifactId>hudi-timeline-service</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
```

Comment:
wondering if this is needed here, at all?

Reply:
here is what is making this necessary. 
We add HoodieAwsconfig in HoodieWriteconfig and hence client-common has to depend on hudi-aws module. If we can follow something like kafka-connect-configs, we can keep HoodieAwsConfig separate and not add it to HoodieWriteConfig, we should be able to get rid of this dep.